server.js

```diff
@@ -0,0 +1,98 @@
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+const databaseHandler = require('./database.js');
+
+app.set('view engine', 'ejs');
+app.use(bodyParser.urlencoded({extended: true}));
+
+databaseHandler.setup( (err, res) => {
+	if(err) return console.log(err);
+	app.listen(3000, () => {
+	console.log('listening on port 3000');
+	console.log('any information inserted into this program will be removed when the program stops');
+	});	
+})
+
+
+app.get('/', (req, res) => {
+	res.sendFile(__dirname + '/views/index.html');
+});
+
+app.get('/insertForm.html', (req, res) => {
+	res.sendFile(__dirname + '/views/insertForm.html');
+});
+
+app.get('/updateForm.html', (req, res) => {
+	res.sendFile( __dirname + '/views/updateForm.html' );
+});
+
+app.get('/deleteForm.html', (req, res) => {
+	res.sendFile( __dirname + '/views/deleteForm.html');
+});
+
+app.get('/search.html', (req, res) => {
+	res.sendFile(__dirname +'/views/search.html');
+})
```

Comment:
For the static files, is there a way to simply serve all static files, instead of manually specifying.